src/flask_restalchemy/resources/querybuilder.py

```diff
@@ -71,6 +72,9 @@ def build_filter_operator(column_name, request_filter, serializer):
         for field in fields:
             field_name = field.lstrip("-")
             column = getattr(model_class, field_name)
+            if isinstance(column, AssociationProxyInstance):
+                res_query = res_query.outerjoin(column.target_class)
```

Comment:
I think this worth a comment above this line: "join with the associated table and define `column` as the associated property to support sorting"